lms/static/sass/course/modules/_student-notes.scss

```diff
@@ -149,14 +149,20 @@ $notes-annotator-background-dark: rgba(122,122,122,0.6); // taken from annotator
-        background-position: 0 0;
+        width: 16px;
+        height: 16px;
+        content: " ";
+        background: $gray-l5;
+        border-bottom: 1px solid $gray-l2;
+        border-right: 1px solid $gray-l2;
+        -webkit-transform: rotate(45deg);
```

Comment:
NIT: please use the Bourbon Transform (http://bourbon.io/docs/#transform) mixin here.